### PR TITLE
Resolves #1204:Center Images on mobile

### LINKED ISSUE #1204: Center Images on mobile
Page : Home 
designbug

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes #1204 

Changes: Added center tag to image in events.hbs

Screenshots for the change:
screenshot from 2017-08-18 00-16-34
screenshot from 2017-08-18 00-16-56

### CODE REVIEW

**Comment by agbilotia1998** [comment]
Okay @sudheesh001 I will change the center tag to its alternative
Screenshot of full screen laptop
screenshot from 2017-08-18 10-38-21

**Comment by agbilotia1998** [comment]
Okay I will do that.

**Comment by agbilotia1998** [comment]
Okay @Princu7 , thanks.

**Comment by Princu7** [comment]
@agbilotia1998 The site looks nice. Good work! Just a small issue. The indentation of the image tags are of 4 spaces. Please make it equal to 2. Thanks!
```
{{if photo}}
    <img onError="this.onerror=null;this.src='./images/avatar.png';" data-original="{{{photo}}}" class="lazy background-image center" alt="{{{name}}}">
```

**Comment by geekyd** [comment]
@Princu7 Should the  npm-shrinkwrap change?

File: src/backend/_scss/application.scss [hunk 1]
```diff
@@ -2097,3 +2097,7 @@ a.skip:hover {
   margin-bottom: 5px;
   margin-left: 5px;
 }
+
+.center{
+  display: block;
+}
```
Comment (sudheesh001) [inline src/backend/_scss/application.scss]: New line?

**Comment by Princu7** [comment]
Yes. `Folder-hash` is being used. I guess that when I added it, the shrinkwrap did not get updated that time. So, when @agbilotia1998 did `npm install`, they got updated.

**Comment by sudheesh001** [comment]
squash commits into one please.

**Comment by agbilotia1998** [comment]
@Princu7 @sudheesh001 Done, please review :)

File: src/backend/_scss/application.scss [hunk 2]
```diff
@@ -2097,3 +2097,7 @@ a.skip:hover {
   margin-bottom: 5px;
   margin-left: 5px;
 }
+
+.center{
+  display: block;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: @agbilotia1998 the class name is `.center` but it is used for `display` property?

Comment (agbilotia1998) [inline src/backend/_scss/application.scss]: @aayusharora I didn't write the class name according to its properties but just the functionality it is serving , should it be changed to display or something else please mention?

Comment (aayusharora) [inline src/backend/_scss/application.scss]: What is the default `display` of this div.

Comment (agbilotia1998) [inline src/backend/_scss/application.scss]: @aayusharora the default display is inline.

Comment (aayusharora) [inline src/backend/_scss/application.scss]: The class name should be `display-block` then

**Comment by agbilotia1998** [comment]
@aayusharora  @Princu7 @sudheesh001 done,please review :).

**Comment by agbilotia1998** [comment]
@Princu7 done please review :).

**Comment by Princu7** [comment]
There should be a line at the end of the `application.scss file`. It was there before but it isn't there anymore. Please see. Thanks!

**Comment by Princu7** [comment]
Just squash the commits! Thanks!

**Comment by Princu7** [comment]
@aayusharora @sudheesh001 Please review. It's been quite some time since it is opened :) Thanks!

**Pull request merged**